### ISSUE TITLE
Overriding a required parameter causes schema error

### ISSUE THREAD

**Issue by romkhub** [anttiviljami/openapi-backend#46]
OAS 3 allows overriding parameters at operation level. I noticed that overriding a required parameter gives error as long as I keep it required at both path-level and operation-level.
https://swagger.io/docs/specification/describing-parameters/#common-for-path:
> Specific path-level parameters can be overridden on the operation level, but cannot be removed.

\- there is an example with override, and here is a gist with that example, producing error - https://gist.github.com/romkhub/71ec632d37d1e1b4057ca652c82d3947 - error is
```
Error: schema is invalid: data.properties['path'].required should NOT have duplicate items (items ## 1 and 0 are identical)
```

**Comment by anttiviljami** [comment]
Looks like a bug. Will have a look

**Comment by anttiviljami** [comment]
Fixed in version `2.3.8` !
